examples/cartpole/README.md

```diff
@@ -1,34 +1,65 @@
 
-## Contents
-- [CartPole discrete](https://github.com/aidudezzz/deepworlds/tree/dev/examples/cartpole/cartpole_discrete)
-- [CartPole continuous](https://github.com/aidudezzz/deepworlds/tree/dev/examples/cartpole/cartpole_continuous)
+----
 
+### Discrete action space
 
-## Solved CartPole - PPO discrete
+The discrete action space case is solved with a custom implementation of the
+[Proximal Policy Optimization Reinforcement (PPO) Learning (RL) algorithm](https://openai.com/blog/openai-baselines-ppo/), 
+both with the emitter - receiver 
+([supervisor script](./cartpole_discrete/controllers/supervisor_manager/supervisor_controller.py), [robot script](./cartpole_discrete/controllers/robot_controller/robot_controller.py)) and 
+robot supervisor ([robot - supervisor script](./cartpole_discrete/controllers/robot_supervisor_manager/robot_supervisor.py)) schemes. 
 
-Trained agent showcase:
+Moreover, a solution using 
+[StableBaselines3 is provided](./cartpole_discrete/controllers/robot_supervisor_manager_Stable_Baselines/PPO_runner.py). 
 
-![image](https://github.com/aidudezzz/deepworlds/blob/dev/examples/cartpole/doc/gif/cartpoleSolved.gif)
+You can find the corresponding .wbt world files to open in Webots [here](./cartpole_discrete/worlds/).
+
+----
+
+### Continuous action space
+
+The continuous action space case is solved with a custom implementation of the 
+[Deep Deterministic Policy Gradient RL algorithm](https://arxiv.org/abs/1509.02971), 
+both with the emitter - receiver 
+([supervisor script](./cartpole_continuous/controllers/supervisor_manager/supervisor_controller.py), [robot script](./cartpole_continuous/controllers/robot_controller/robot_controller.py)) and 
+robot supervisor ([robot - supervisor script](./cartpole_continuous/controllers/robot_supervisor_manager/robot_supervisor.py)) schemes. 
+
+You can find the corresponding .wbt world files to open in Webots [here](./cartpole_continuous/worlds/).
+
+----
+
+### Contents
+- [CartPole discrete](./cartpole_discrete)
+  - [supervisor](./cartpole_discrete/controllers/supervisor_manager/) / [robot](./cartpole_discrete/controllers/robot_controller/) / [custom PPO](./cartpole_discrete/controllers/supervisor_manager/agent/PPO_agent.py)
+  - [robot - supervisor](./cartpole_discrete/controllers/robot_supervisor_manager) / [custom PPO](./cartpole_discrete/controllers/robot_supervisor_manager/agent/PPO_agent.py)
+  - [robot - supervisor with StableBaselines3](./cartpole_discrete/controllers/robot_supervisor_manager_Stable_Baselines/)
+- [CartPole continuous](./cartpole_continuous)
+  - [supervisor](./cartpole_continuous/controllers/supervisor_manager/) / [robot](./cartpole_continuous/controllers/robot_controller/) / [custom DDPG](./cartpole_continuous/controllers/supervisor_manager/agent/DDPG_agent.py)
+  - [robot - supervisor](./cartpole_continuous/controllers/robot_supervisor_manager) / [custom DDPG](./cartpole_continuous/controllers/robot_supervisor_manager/agent/DDPG_agent.py)
```

Comment:
@tsampazk 
When users see  `/`, users might think `robot (./controllers/robot_controller/)` is in `supervisor (.//controllers/supervisor_manager/)` folder.
I have no idea how to make it better, though. 😕

Reply:
Oh sorry i didn't get it the first time you mentioned that. I used / as a separator, i can go ahead and change it to a ',' or '-', what do you think?

Reply:
I replaced them with commas, let me know how it looks.

Reply:
@tsampazk Thanks! LGTM 😄